tests/regression/test_permissions.py

```diff
@@ -4,164 +4,222 @@
+                "STAKING_MODULE_PAUSE_ROLE": [contracts.deposit_security_module],
+                "STAKING_MODULE_RESUME_ROLE": [contracts.deposit_security_module],
+                "STAKING_MODULE_MANAGE_ROLE": [],
+                "REPORT_EXITED_VALIDATORS_ROLE": [contracts.accounting_oracle],
+                "UNSAFE_SET_EXITED_VALIDATORS_ROLE": [],
+                "REPORT_REWARDS_MINTED_ROLE": [contracts.lido],
+            },
+        },
+        "WithdrawalQueue": {
+            "contract": contracts.withdrawal_queue,
+            "type": "CustomApp",
+            "proxy_owner": contracts.agent,
+            "roles": {
+                "DEFAULT_ADMIN_ROLE": [contracts.agent],
+                "PAUSE_ROLE": [gate_seal_address],
+                "RESUME_ROLE": [],
+                "FINALIZE_ROLE": [contracts.lido],
+                "ORACLE_ROLE": [contracts.accounting_oracle],
+                "MANAGE_TOKEN_URI_ROLE": [],
+            },
+        },
+        "AccountingOracle": {
+            "contract": contracts.accounting_oracle,
+            "type": "CustomApp",
+            "proxy_owner": contracts.agent,
+            "roles": {
+                "DEFAULT_ADMIN_ROLE": [contracts.agent],
+                "SUBMIT_DATA_ROLE": [],
+                "MANAGE_CONSENSUS_CONTRACT_ROLE": [],
+                "MANAGE_CONSENSUS_VERSION_ROLE": [],
+            },
+        },
+        "ValidatorsExitBusOracle": {
+            "contract": contracts.validators_exit_bus_oracle,
+            "type": "CustomApp",
+            "proxy_owner": contracts.agent,
+            "roles": {
+                "DEFAULT_ADMIN_ROLE": [contracts.agent],
+                "SUBMIT_DATA_ROLE": [],
+                "PAUSE_ROLE": [gate_seal_address],
+                "RESUME_ROLE": [],
+                "MANAGE_CONSENSUS_CONTRACT_ROLE": [],
+                "MANAGE_CONSENSUS_VERSION_ROLE": [],
+            },
+        },
+        "AccountingHashConsensus": {
+            "contract": contracts.hash_consensus_for_accounting_oracle,
+            "type": "CustomApp",
+            "roles": {
+                "DEFAULT_ADMIN_ROLE": [contracts.agent],
+                "MANAGE_MEMBERS_AND_QUORUM_ROLE": [],
+                "DISABLE_CONSENSUS_ROLE": [],
+                "MANAGE_FRAME_CONFIG_ROLE": [],
+                "MANAGE_FAST_LANE_CONFIG_ROLE": [],
+                "MANAGE_REPORT_PROCESSOR_ROLE": [],
+            },
+        },
+        "ValidatorsExitBusHashConsensus": {
+            "contract": contracts.hash_consensus_for_validators_exit_bus_oracle,
+            "type": "CustomApp",
+            "roles": {
+                "DEFAULT_ADMIN_ROLE": [contracts.agent],
+                "MANAGE_MEMBERS_AND_QUORUM_ROLE": [],
+                "DISABLE_CONSENSUS_ROLE": [],
+                "MANAGE_FRAME_CONFIG_ROLE": [],
+                "MANAGE_FAST_LANE_CONFIG_ROLE": [],
+                "MANAGE_REPORT_PROCESSOR_ROLE": [],
+            },
+        },
```

Comment:
Can we add a check for the committee members as it's done for the DSM guardians?